src/grandorgue/model/pipe-config/GOPipeConfig.h

```diff
@@ -37,6 +37,7 @@ class GOPipeConfig {
   float m_PitchCorrection;
   float m_ManualTuning;
   float m_AutoTuningCorrection;
+  int m_ToneBalanceValue;
```

Comment:
```suggestion
  int8_t m_ToneBalanceValue;
```

Snd please move it after m_ReleaseTail